recipes-devtools/python/python3-ovs_%.bbappend

```diff
@@ -0,0 +1,4 @@
+# Upstream-Status: unsubmitted
```

Comment:
I really like these Upstream-Status lines. But now I wonder, why has it not been submitted?

Reply:
Because upstreaming takes additional time and effort (and first maybe we want to make sure it works/does its thing)